pkg/graph/schema.resolvers.go

```diff
@@ -81,8 +81,6 @@ func (r *mutationResolver) CreateModelPlan(ctx context.Context, input model.Mode
 
 	plan := ConvertToModelPlan(&input)
 
-	plan.CreatedBy = &principal
```

Comment:
thanks for making this consistent! I think this is the paradigm that we should follow.

Reply:
Agree. I think we may even introduce some generic "always modify created by, modified by" logic somewhere that's shared, as I think this is readable, but probably easy to forget to add to a resolver!